syntax = "proto3";

service TalkToMe {
    rpc HelloWorld (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string msg = 2;
}
